java/arcs/core/data/expression/EvaluatorParticle.kt

```diff
@@ -190,6 +190,7 @@ private abstract class SchemaGuidedTranslator {
     is FieldType.Primitive -> value
     is FieldType.EntityRef -> TODO("References are not supported")
     is FieldType.Tuple -> TODO("Tuples are not supported")
+    is FieldType.NullableOf -> translateSingleton(type.innerType, value)
```

Comment:
Are nullable types fully supported by paxel yet? If not, maybe this should just be a TODO as well.

Otherwise, test please :)

Reply:
Paxel has untagged type unions and a null type, so the manifest's `T?` becomes `T | NullType`

I'll make this a TODO though and add another PR to impl Paxel related code + testing